generation/core/generator.js

```diff
@@ -107,13 +107,12 @@ module.exports = function getGenerator({
 
     const overloadFunctionExpressions = json.instances.map(({ args }) => {
       const params = [...(json.prefixedArgs || []), ...args];
-        return t.functionDeclaration(
-          t.identifier(sanitizedName + params.length),
-          params.filter(filterBlacklistedArguments).map(({ name }) => t.identifier(name)),
-          createMethodBody(classJson, Object.assign({}, json, { args }))
-        )
-      }
-    );
+      return t.functionDeclaration(
```

Comment:
While the overload generation is a brilliant mechanism and this is worth its own pull request, can we stop for a a second and discuss if a DTO argument could be a better idea, still?

Reply:
I want to align with the current `longPress(duration)` API we have (iOS-only)

Reply:
we can, possibly, enable both DTO and `duration` only but it feels a bit dirty.
also, it makes sense to align with `tap({x: number, y: number})` for the point param.